crates/store/re_types/src/archetypes/asset_video_ext.rs

```diff
@@ -39,6 +39,24 @@ impl AssetVideo {
         }
     }
 
+    /// Returns the bytes of the video blob.
+    pub fn video_blob_bytes(&self) -> Option<&[u8]> {
+        let blob_list_array = self
+            .blob
+            .as_ref()?
+            .array
+            .as_any()
+            .downcast_ref::<arrow::array::ListArray>()
+            .expect("Video blob data is not a ListArray");
+        let blob_data = blob_list_array
+            .values()
+            .as_any()
+            .downcast_ref::<arrow::array::PrimitiveArray<arrow::datatypes::UInt8Type>>()
+            .expect("Video blob data is not a PrimitiveArray<UInt8>");
+
+        Some(blob_data.values().inner().as_slice())
```

Comment:
Wouldn't most of this code make more sense in `blob_ext.rs`? So that:

A) it is closer to the logic that controls how it is encoded
B) it can be reused by other archetypes that also has `Blob` in them

Reply:
the only hurdle is that we don't have `Blob` type here. But we can just make one up on the spot 🤷

Reply:
right, there's no blob archetype.
We can have a static method on the blob datatype to help with this kind of thing, but we're dealing with serialized blob data here whereas `Blob` component and `Blob` datatype are about unseralized data